test/openAPI/features/support/wallet_generate_link_auth_code.js

```diff
@@ -187,6 +231,150 @@ When(
             })
 );
 
+// Scenario: Not able to validate the link-code and its expiry because of transaction and link code are not connected to each other
+// Given and others Then for this scenario are written in the aforementioned example
+
+When(
+    /^Send POST \/linked\-authorization\/link\-auth\-code request with given valid linkCode and transactionId$/,
+    () =>
+        specWalletGenerateLinkAuthCode.post(baseUrl)
+            .withHeaders(X_XSRF_TOKEN.key, X_XSRF_TOKEN.value)
+            .withJson({
+              requestTime: new Date().toISOString(),
+              request: {
+                linkedCode: 'valid_link_code_001',
+                transactionId: transactionId
+              },
+            })
+);
+
+// Scenario: Not able to validate the link-code and its expiry because of reuse of the completed transaction_id
+// Given and others Then for this scenario are written in the aforementioned example
+
+Given(
+    /^The first authorization flow for transactionId ends$/,
+    async () => {
+      specWalletGenerateLinkCode
+          .post(localhost + walletGenerateLinkCodeEndpoint)
+          .withHeaders(X_XSRF_TOKEN.key, X_XSRF_TOKEN.value)
+          .withJson({
+            requestTime: new Date().toISOString(),
+            request: {
+              transactionId: transactionId,
+            },
+          });
+      await specWalletGenerateLinkCode.toss();
+
+      receivedLinkCode =
+          specWalletGenerateLinkCode._response.json.response.linkCode;
+
+      specWalletLinkTransaction
+          .post(localhost + walletLinkTransactionEndpoint)
+          .withJson({
+            requestTime: new Date().toISOString(),
+            request: {
+              linkCode: receivedLinkCode,
+            },
+          });
+
+      await specWalletLinkTransaction.toss();
+
+      receivedLinkedTransactionId =
+          specWalletLinkTransaction._response.json.response.linkTransactionId;
+
+      specWalletLinkedAuthenticate.post(localhost + walletLinkedAuthenticateEndpoint)
+          .withJson({
+            requestTime: new Date().toISOString(),
+            request: {
+              linkedTransactionId: receivedLinkedTransactionId,
+              individualId: individualId,
+              challengeList: [
+                {
+                  authFactorType: "PIN",
+                  challenge: "password",
+                  format: "alpha-numeric",
+                },
+              ],
+            },
+          })
+
+      await specWalletLinkedAuthenticate.toss();
+
+      specWalletGenerateLinkAuthCode.post(baseUrl)
+          .withHeaders(X_XSRF_TOKEN.key, X_XSRF_TOKEN.value)
+          .withJson({
+            requestTime: new Date().toISOString(),
+            request: {
+              linkedCode: receivedLinkCode,
+              transactionId: 'transaction_id_004'
```

Comment:
we cannot use hardcoded transaction id, we have to use the same transaction id used for generating the link code